### DISCUSSION TITLE
HeyForm Branding Removal

### DISCUSSION THREAD

**Discussion by asrulmunir** [heyform/heyform#132] [Self hosting]
Hi. 

Is there any restriction to remove HeyForm branding for the community version?

If no, how can we do this?

TIA.

**Comment by asrulmunir** [comment]
IMHO, since the community version has less features compared to the cloud-hosted version, branding removal should be an option to make HeyForm more desirable for people to try and can start more word of mouth on how awesome HeyForm is 👌

**Comment by graphiostudio** [comment]
@asrulmunir this is just another one of those "_open source_" projects that gives you almost nothing. Good for marketing, though. You'll deploy yourself and realize you can't do much with it, no integrations, not a single setting anywhere, it's just a form that you can actually create yourself if you have the knowledge to deploy this in the first place. They're asking $29/month to remove the branding mate, do you think they're going to let you just remove it for free? 
'But it's **open sourceeeee**, you can deploy on your own server and domain'    yeaahhhhh suuuure!!!

**Reply by JOduMonT** [reply]
Open Source does not mean Free;
and Free doesn't mean Open Source.

Open Source developers need to eat too.

**Comment by gelbermoran** [comment]
.heyform-branding {
  display: none;
  visibility: hidden;
}

**Comment by JOduMonT** [comment]
> Is there any restriction to remove HeyForm branding for the community version?

HeyForm use [AGPL-3](https://github.com/heyform/heyform?tab=AGPL-3.0-1-ov-file)
### which give you the right to modify it.
> Modify the code: Users can change or rework the code, but if they distribute these changes/modifications in any public way (including over a server), they must release these updates in source code form under the AGPL license.

**Comment by ScrumpyJack** [comment]
This is what I put in the custom css section when I do simple single page feedback forms


/* Hide question number */
.heyform-block-number {
  display: none;
}

/* Hide password and abuse warning */
.heyform-submit-warn {
  display: none;
}

/* Hide branding */
.heyform-branding {
  display: none;
}


/* Hide Previous Next Buttons */
.heyform-footer-right {
  display: none;
}
